src/runtime/tests/test-environment-test.ts

```diff
@@ -7,17 +7,20 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-import {registerSystemExceptionHandler, PropagatedException} from '../arc-exceptions.js';
+import {registerSystemExceptionHandler, PropagatedException, removeSystemExceptionHandler, defaultSystemExceptionHandler} from '../arc-exceptions.js';
 
-let exceptions: PropagatedException[] = [];
+let exceptions: Error[] = [];
 
-beforeEach(() => registerSystemExceptionHandler((exception) => exceptions.push(exception)));
+before(() => {
+  removeSystemExceptionHandler(defaultSystemExceptionHandler);
+  registerSystemExceptionHandler((exception) => exceptions.push(exception));
```

Comment:
Same comments here as in the file above (can you just de-dupe these files?)

Reply:
Alas, no, unless we were to roll the deduplication into sigh test generation, which I don't really want to tackle right now.